html_tag_collector/collector.py

```diff
@@ -264,7 +265,13 @@ def parse_response(url_response):
     except (bs4.builder.ParserRejectedMarkup, AssertionError, AttributeError):
         return tags
 
-    tags["html_title"] = soup.title.string.strip() if soup.title is not None else ""
+    if soup.title is not None:
+        if soup.title.string is not None:
+            tags["html_title"] = soup.title.string.strip()
+        else:
+            tags["html_title"] = ""
+    else:
+        tags["html_title"] = ""
```

Comment:
This logic can also be simplified -- we don't need two else clauses performing the same thing if we join the if conditionals with an and.

```python
if soup.title is not None and soup.title.string is not None:
    tags["html_title"] = soup.title.string.strip()
else:
    tags["html_title"] = ""
```